grocery-store.rb

```diff
@@ -0,0 +1,117 @@
+#class Cusotmer
```

Comment:
You don't really need this file, you should have:
In the `lib` folder:
*  `order.rb` 
*  `customer.rb`

And use the corresponding specs in the `specs` folder.